tools/generate-changelog.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2019 Canonical Ltd
```

Comment:
Was this copied from somewhere?

Reply:
yes - it's a modified copy of the one from the system snaps CICD repo

Reply:
but good point about the copyright year

Reply:
But what is the strategy here. if we fix anything here. Should we also fix it somewhere else?

Reply:
Please keep the 2019 as well, if it was modified.

Reply:
The strategy here is to just copy it per repo for now - once we spend some time to streamline our setup to make the base repos use the same CICD repo as the system-snaps, we will consolidate the support for changelogs there.

The script is small, relatively simple and specialized, it does (IMHO) not make sense to create a new repo for just that *or* pull in the entire CICD repo to have a variant of the script there.

Reply:
If you or  @alfonsosanchezbeato feel strongly about the alternative (put it into cicd and pull it in), i will happily change it

Reply:
I do not care that much. But it would be nice to say where it originally came from. Just because if it stops working, we can see if the other place already has some fix.

Reply:
And it would be nice also to say the hash of the commit it was you copied from.

Reply:
@valentindavid I added a link to the original script and the hash, i think its good suggestions